mux.go

```diff
@@ -52,13 +52,13 @@ func NewMux(opt *MuxOptions) *chi.Mux {
 	r := chi.NewMux()
 	r.Use(middleware.RequestID)
 	r.Use(middleware.RealIP)
-	r.Use(middleware.RequestLogger(o.Logger))
 	r.Use(middleware.Recoverer)
 
 	if o.Heartbeat != "false" {
 		r.Use(middleware.Heartbeat(o.Heartbeat))
 	}
 
+	r.Use(middleware.RequestLogger(o.Logger))
```

Comment:
Are you sure it should be BEFORE the Recoverer? I'd prefer to see 5xx errors logged on panics.

Maybe it would be to just move Heartbeat to the very top of middleware stack instead?

Reply:
https://github.com/bsm/httpx/pull/5